ls1intum/Artemis
# Feature/course color extension

### Checklist
- [x] I tested *all* changes and *all* related features on the test server https://artemistest.ase.in.tum.de.

### Motivation and Context


Additions regarding the courses' colors. 

### Description

Added a column visualizing the color of each course (where the color is set) at the beginning of each course row in the Course Administration.
Added multiple colors to the list of default colors that are displayed in the window with colors when creating a new course.  
Adapted the height of the window.

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. At the beginning of each course row you should now be able to see the course's color
4. Click on create new course 
5. In the Color field you should be able to see the new colors

### Screenshots 

![Create new course](https://user-images.githubusercontent.com/33368508/67322274-12ca2e80-f511-11e9-8cda-4d57822cb7cd.png)

![Course administration](https://user-images.githubusercontent.com/33368508/67322291-165db580-f511-11e9-832f-218d97e56f90.png)
